regression/Makefile

```diff
@@ -15,6 +15,7 @@ DIRS = ansi-c \
        goto-instrument-typedef \
        invariants \
        strings \
+       jbmc \
```

Comment:
Does `clobber` need to be added to this list too?

Reply:
And this needs to be added to the cmake list file as well

Reply:
clobber is not compiled by default. Adding it to the default regression test list would cause a test failure.

Reply:
Not sure if there is much value in having tests that aren't run... is the plan to split it out into a separate repository at some point?